scripts/run-test.ts

```diff
@@ -64,7 +65,7 @@ const main = async () => {
 					.array(
 						z
 							.string()
-							.refine((value) => value.match(/^(?:\w+(?:-\w+)*|\*)$/))
+							.refine((value) => value.match(/^(?:[.\w]+(?:-[.\w]+)*|\*)$/))
```

Comment:
The `fixturesInclude` and `fixturesExclude` regex should be the same, right?

Reply:
sorry, yeah I was tweaking them and one was unintentionally left with an older attempt 😓